src/Components/Cart.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+
+const Cart = ({shoppingCart, total, tax, subtotal}) => {
```

Comment:
props are case sensitive. You pass a prop named 'subTotal' inside the App component and try to reference it here as 'subtotal'.